src/App.js

```diff
@@ -1,25 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculate from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Based on project requirements the App.js component should be class-based. Because before using functions in the following projects we have to understand how to use classes 
![Screenshot (138)](https://user-images.githubusercontent.com/67087664/150175346-aa56e8b0-b0b1-4609-ad87-3a76f024b81e.png)

- [ ] Also, I am requesting one more change here because there is no `public` folder in your `files changed`. In your project directory, there is a `public `folder and there are some unnecessary files such as `images `and `texts`. I recommend removing them in order to make your codebase clean.